Source/WebCore/page/LocalFrame.h

```diff
@@ -155,6 +155,7 @@ class LocalFrame final : public Frame, public CanMakeCheckedPtr {
     FrameLoader& loader() { return m_loader.get(); }
     CheckedRef<const FrameLoader> checkedLoader() const;
     CheckedRef<FrameLoader> checkedLoader();
+
```

Comment:
What's the deal with this blank line?

Reply:
I was trying to separate the 4 loader getters from the rest. They already had a blank line before them. It thought it looked nicer this way.